### PR TITLE
Adds Mariadb supports

### PR DESCRIPTION
**Pull request by rolancia**

### CODE REVIEW

**Comment by rolancia** [comment]
@300481

**Comment by 300481** [comment]
Hi @rolancia, thank you very much for your contribution efforts!

I already had this dependency in my chart and a few versions ago I decided to remove it.
Why? It is an additional dependency to maintain and keep it updated.
I decided to deploy the database not within the chart and losely couple it with just a database credential secret.
This will allow you to use any database deployment method you are familiar with,
e.g. the Bitnami Chart or a MariaDB operator, which keeps the database updated.

To summarize it, first I want to technically decouple it, second I want to free up my limited time to maintain the chart.

Sorry to say that, but will not merge your request. Maybe you can create your own umbrella-chart or,
as I do, deploy it by an ArgoCD Application.

**Comment by rolancia** [comment]
understand. thank you for your explaining

but it could be a good option to add this for light(non-expert) users if you want to support them.
when i met this chart to compare seafile with other services, i expected to work when i just helm installed but not. (as Owncloud, Nextcloud, many service charts provide)
@300481
